2.7/Dockerfile

```diff
@@ -55,10 +55,16 @@ RUN INSTALL_PKGS="libjpeg-turbo libjpeg-turbo-devel python27 python27-python-dev
     yum -y clean all --enablerepo='*'
 
 # Copy the S2I scripts from the specific language image to $STI_SCRIPTS_PATH.
-COPY ./s2i/bin/ $STI_SCRIPTS_PATH
+COPY 2.7/s2i/bin/ $STI_SCRIPTS_PATH
 
 # Copy extra files to the image.
-COPY ./root/ /
+COPY 2.7/root/ /
+
+# Yes, the directory below is already copied by the previous command.
+# The problem here is that the wheels directory is copied as a symlink.
+# Only if you specify symlink directly as a source, COPY copies all the
+# files from the symlink destination.
+COPY 2.7/root/opt/wheels /opt/wheels
```

Comment:
`/opt/` is kind of a free for all, so I'm thinking whether we shouldn't choose some longer directory name, to decrease likelihood of any possible conflicts in the future? Or maybe I'm being too paranoid.

Reply:
Something like `__wheels`?

Reply:
How about `__wheels__`? :D

Reply:
Is there a reason for not using `/tmp` considering we are removing the directory shortly after?

Reply:
@pkubatrh Good question. However, that makes me question if we should be even removing it. I think we're shipping the images not squashed, is that right? If so, we could just as well keep the file in the container, for debugging and non-standard use cases.

Reply:
rhel-based images are definitely getting squashed. Not sure about centos images now that we are moving to quay.io. @phracek might know

Reply:
Ah, ok, if that's so, Ii agree that /tmp is not a bad choice.

Reply:
Switched to `/tmp/wheels/`.

Reply:
I guess CentSO7 images are squashed. https://quay.io/repository/centos7/python-27-centos7?tab=builds